app/services/foods_service.rb

```diff
@@ -0,0 +1,16 @@
+class FoodsService
+  def conn
+    Faraday.new(url: "https://api.nal.usda.gov/fdc/") do |f|
+      f.headers["X-Api-Key"] = Rails.application.credentials.fdc[:key]
+    end
+  end
+
+  def get_url(url)
+    response = conn.get(url)
+    JSON.parse(response.body, symbolize_names: true)
+  end
+
+  def search_foods(search_value)
+    get_url("v1/foods/search?query=#{search_value}&pageSize=10")
```

Comment:
Really good service call to only get the first 10 that you need 👍  Not every API service will allow this, so consider another way you could limit your response to 10 just for good practice.